sdk/iot/provisioning/inc/az_iot_provisioning_client.h

```diff
@@ -228,16 +233,19 @@ typedef struct az_iot_provisioning_client_registration_state
 typedef struct az_iot_provisioning_client_register_response
 {
   az_iot_status status; /**< The current request status.
-                         * @note The authoritative response for the device registration operation
+                         * @remark The authoritative response for the device registration operation
                          * (which may require several requests) is available only through
-                         * #registration_state.  */
+                         * #operation_status.  */

```

Comment:
I think this should be backticks. `operation_status` isn't a type and won't resolve.

Reply:
Right: I'm going to fix all of them (searching for # in generated html).